support-frontend/app/wiring/Controllers.scala

```diff
@@ -77,6 +77,22 @@ trait Controllers {
     fontLoader
   )
 
+  lazy val weeklyController = new WeeklySubscription(
+    authAction,
```

Comment:
this is a lot of parameters for one class! surely every method can't need all of them 🤔